### ISSUE TITLE
Godot collects and processes clicks that are not for him.

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#50761]
### Godot version

3.3.2 stable mono

### System information

w10 64

### Issue description

Godot activates the audio panel even though I did not click on the panel activation. The click was directed to maximize chrome from the taskbar.

Watch the video, the click is made on the chrome icon in the taskbar but godot activates the audio panel.

https://user-images.githubusercontent.com/19652075/126755140-14f7ef96-0cf4-4992-81fc-3af876c2493a.mp4

### Steps to reproduce

With the editor maximized and activated (select some object or something in the editor window as seen in the video). Then select an icon that matches the activation section of some godot panel and check if the panel in godot is activated.

**Comment by Calinou** [comment]
I don't see how this could be happening. My guess is that the auto-hiding taskbar is finnicky and occasionally lets clicks go through it. In this case, there is nothing Godot can do about this. Consider using <kbd>Alt + Tab</kbd> / <kbd>Alt + Shift + Tab</kbd> more often.

**Comment by Torguen** [comment]
Yes, well I like to hide it automatically because that way I get more space for applications.

Indeed, if the bar is not hidden automatically then there is no overlapping of the spaces.
![Captura](https://user-images.githubusercontent.com/19652075/126765313-718835a5-d25c-44f1-a2d0-cc6ba0a2f1ab.PNG)

Ok thanks for the information.
